pkg/eosclient/eosgrpc/eosgrpc.go

```diff
@@ -446,15 +483,17 @@ func (c *Client) GetFileInfoByInode(ctx context.Context, uid, gid string, inode
 		info.Inode = inode
 	}
 
+	log.Info().Str("func", "GetFileInfoByInode").Uint64("inode", inode).Msg("grpc response")
```

Comment:
Change log level to debug. Ditto for all the places where you're logging details about the response

Reply:
Ok. This will do in a comprehensive logging tour